src/System.Private.Xml/src/System/Xml/Serialization/ReflectionXmlSerializationReader.cs

```diff
@@ -14,8 +14,15 @@
 using System.Xml.Extensions;
 using System.Xml.Schema;
 
+#if XMLSERIALIZERGENERATOR
+namespace Microsoft.XmlSerializer.Generator
+#else
 namespace System.Xml.Serialization
+#endif
 {
+    using System.Xml;
+    using System.Xml.Serialization;
```

Comment:
I will stop comment on this :)  Pease put all using together in this file and others.